assembl/alembic/versions/180022eb0f0d_add_extract_hash.py

```diff
@@ -0,0 +1,37 @@
+"""add_extract_hash
+
+Revision ID: 180022eb0f0d
+Revises: 8d37745a8e69
+Create Date: 2018-08-09 23:18:03.815496
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '180022eb0f0d'
+down_revision = '8d37745a8e69'
+
+from alembic import context, op
+import sqlalchemy as sa
+import transaction
+
+
+def upgrade(pyramid_env):
+    with context.begin_transaction():
+        op.add_column(
+            'extract',
+            sa.Column(
+                'extract_hash', sa.String,
```

Comment:
Why do you prefix the field by `extract_`, there is no need.

Reply:
hash is a python keyword